samples/matter/common/src/bridge/bridged_device.cpp

```diff
@@ -22,6 +21,20 @@ CHIP_ERROR BridgedDevice::CopyAttribute(void *attribute, size_t attributeSize, v
 	return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR BridgedDevice::HandleWriteDeviceBasicInformation(chip::ClusterId clusterId, chip::AttributeId attributeId,
+							    void *data, size_t dataSize)
+{
+	switch (attributeId) {
+	case Clusters::BridgedDeviceBasicInformation::Attributes::Reachable::Id:
+		if (data) {
+			SetIsReachable(*reinterpret_cast<bool*>(data));
```

Comment:
It would be good to check if `dataSize` is equal to the `sizeof(bool)`, as it is a size of the whole data buffer and in some fault cases it may be different than expected, what means there might be a garbage instead of normal data. For all other cases I used `CopyAttribute` helper method that verifies it.